Foody/Onboarding/Login/LoginViewController.swift

```diff
@@ -6,29 +6,109 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var loginImageView: UIImageView!
+    
+    @IBOutlet weak var loginLbl: UILabel!
+    @IBOutlet weak var welcomeLbl: UILabel!
+    
+    @IBOutlet weak var rememberMeSwitch: UISwitch!
+    
+    @IBOutlet weak var emailTextFld: UITextField!
+    @IBOutlet weak var passwordTextFld: UITextField!
+    
+    @IBOutlet weak var loginBttn: UIButton!
+    @IBOutlet weak var registerBttn: UIButton!
+    
     var clickedRegisterCompletion:(() -> ())?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        rememberMeSwitch.addTarget(self, action: #selector(self.stateChanged), for: .valueChanged)
+                let defaults: UserDefaults = UserDefaults.standard
+                if defaults.bool(forKey: "ISRemember") {
+                    emailTextFld.text = defaults.value(forKey: "SavedUserEmail") as! String
+                    passwordTextFld.text = defaults.value(forKey: "SavedPassword") as! String
+                    rememberMeSwitch.setOn(true, animated: false)
+                    print("already logged user")
+                }
+                else {
+                    rememberMeSwitch
+                        .setOn(false, animated: false)
+                }
+        
+        loginImageView.image = UIImage(named: "chicken")
+        welcomeLbl.text = "Welcome"
+        loginLbl.text =  "Login your account"
+        emailTextFld.layer.cornerRadius = 25
+        passwordTextFld.layer.cornerRadius = 25
+        loginBttn.layer.cornerRadius = 15
+        registerBttn.layer.cornerRadius = 15
     }
-    
+   
+    @objc func stateChanged(_ switchState: UISwitch) {
+           let defaults: UserDefaults? = UserDefaults.standard
+        if switchState.isOn {
+               defaults?.set(true, forKey: "ISRemember")
+               defaults?.set(emailTextFld.text, forKey: "SavedUserEmail")
+               defaults?.set(passwordTextFld.text, forKey: "SavedPassword")
+           }
+           else {
+               defaults?.set(false, forKey: "ISRemember")
+               }
+               }
+    @IBAction func rememberMeSwitchChck(_ sender: Any) {
+
+            }
+    @IBAction func loginBttnClicked(_ sender: Any) {
+        if emailTextFld.text != "" && passwordTextFld.text != "" {
+                   Auth.auth().signIn(withEmail: emailTextFld.text!, password: passwordTextFld.text!){
```

Comment:
Remove !  . Try to find way  to get rid of ! (emailTextFld.text! and  passwordTextFld.text!)